### ISSUE TITLE
Can’t build without GLib

### ISSUE THREAD

**Issue by khaledhosny** [centricular/harfbuzz#2]
Because of #1, if no GLib is found in the system, the build will try to build without GLib and it always fails with:
```
test/shaping/meson.build:87:2: ERROR:  Unknown variable "hb_shape".
```

**Comment by tp-m** [comment]
I'll have a look at this.

**Comment by tp-m** [comment]
Not quite there yet with MSVC and `meson -Dglib=disabled -Dgobject=disabled -Duniscribe=auto -Ddirectwrite=auto -Dfontconfig=disabled`:
```
[51/57] Linking target src/libharfbuzz-0.dll.
FAILED: link @src/libharfbuzz-0.dll.rsp
harfbuzz.def : error LNK2001: unresolved external symbol hb_ft_face_create
harfbuzz.def : error LNK2001: unresolved external symbol hb_ft_face_create_cached
harfbuzz.def : error LNK2001: unresolved external symbol hb_ft_face_create_referenced
harfbuzz.def : error LNK2001: unresolved external symbol hb_ft_font_changed
harfbuzz.def : error LNK2001: unresolved external symbol hb_ft_font_create
harfbuzz.def : error LNK2001: unresolved external symbol hb_ft_font_create_referenced
harfbuzz.def : error LNK2001: unresolved external symbol hb_ft_font_get_face
harfbuzz.def : error LNK2001: unresolved external symbol hb_ft_font_get_load_flags
harfbuzz.def : error LNK2001: unresolved external symbol hb_ft_font_set_funcs
harfbuzz.def : error LNK2001: unresolved external symbol hb_ft_font_set_load_flags
harfbuzz.def : error LNK2001: unresolved external symbol hb_glib_blob_create
harfbuzz.def : error LNK2001: unresolved external symbol hb_glib_get_unicode_funcs
harfbuzz.def : error LNK2001: unresolved external symbol hb_glib_script_from_script
harfbuzz.def : error LNK2001: unresolved external symbol hb_glib_script_to_script
src\\harfbuzz.lib : fatal error LNK1120: 14 unresolved externals
```

**Comment by tp-m** [comment]
Should be fixed now in PR #3

**Comment by khaledhosny** [comment]
Thanks, works fine now (with #4 as I’m building harfbuzz as a subproject).
